packages/common-all/src/dnode.ts

```diff
@@ -1253,6 +1270,13 @@ export class SchemaUtils {
       }),
       type: "schema",
     });
+
+    // @ts-ignore
```

Comment:
since `SchemaOpts` and `SchemaRaw` are distinct, can we split this into two methods? 
- createFromSchemaOpts
- createFromSchemaRaw

That way we can remove the `ts-ignore`

Reply:
Split the `create` into 

```
createFromSchemaOpts
createFromSchemaRaw
```

Quite a few touch points with it, mostly tests though. 

Tests pass, and manual testing within test-workspace related to schema `dendron.schemas` works.